website/docs/api/misc/eslint-plugin/README.md

```diff
@@ -15,26 +15,25 @@ npm install --save-dev @docusaurus/eslint-plugin
 
 ## Usage
 
-Add `@docusaurus` to the plugins section of your `.eslintrc` configuration file:
+### Recommended Config
 
-```json title=".eslintrc"
-{
-  "plugins": ["@docusaurus"]
-}
-```
-
-Then, you can extend one of the configs (e.g. the `recommended` config):
+Add `plugin:@docusaurus/recommended` to the `extends` section of your `.eslintrc` configuration file:
 
 ```json title=".eslintrc"
 {
   "extends": ["plugin:@docusaurus/recommended"]
 }
 ```
 
-Each config contains a set of rules. For more fine-grained control, you can also configure the rules you want to use directly:
+This will enable the `@docusaurus` eslint plugin and use the `recommended` config. See [Supported Rules](#supported-rules) below for a list of rules that this will enable.
```

Comment:
```suggestion
This will enable the `@docusaurus` eslint plugin and use the `recommended` config. See [supported Rules](#supported-rules) below for a list of rules that this will enable.
```